src/Sidebar/Menu/MenuItem.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { MainLink, MenuItem as MenuItemStyled } from './styled';
+
+export default class MenuItem extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { isActive: false };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    this.setState({ isActive: !this.state.isActive });
```

Comment:
https://medium.freecodecamp.com/functional-setstate-is-the-future-of-react-374f30401b6b
https://medium.com/@shopsifter/using-a-function-in-setstate-instead-of-an-object-1f5cfd6e55d1